Documentation/How-to-log-life.md

```diff
@@ -138,6 +138,19 @@ Also the use of audio recordings from chat applications or voice-to-text transcr
 
 All bureaucratic documentation can be saved in PDF format by cell phone, either by photo or by the free program ScanPDF on Android. Psychological and psychiatric tests are also a great source of information or sessions with a psychologist or psychiatrist. After starting this compilation it is very important to classify and sort all the data by format and subject for further conceptual processing by the future artificial intelligence program.
 
+# How to save all information from Anki software.
+
+Anki is a software for spaced repetition, where you place flash cards in a Q-A style for improve the learning.
```

Comment:
```suggestion
Anki is a software for spaced repetition, where you place flash cards in a QnA style for improve the learning.
```

Reply:
All right, I'll commit all your suggestions. Thank you!